src/main/java/fruitshop/dataservice/SupplierReport.java

```diff
@@ -0,0 +1,24 @@
+package fruitshop.dataservice;
+
+import fruitshop.database.Storage;
+import fruitshop.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SupplierReport implements WorkWithReport {
+    private static final String COLUMN_NAMES = "fruit,quantity";

```

Comment:
let's keep names more consistant, let's change WorkWithReport to ReportCreator and SupplierReport to ReportCreatorImpl
check other names